package.json

```diff
@@ -38,39 +38,39 @@
   ],
   "license": "OGL-UK-3.0",
   "dependencies": {
-    "@hapi/hapi": "20.2.2",
-    "@hapi/hoek": "9.1.1",
-    "@hapi/inert": "6.0.3",
-    "@hapi/vision": "6.1.0",
-    "@hapi/wreck": "17.0.0",
-    "@hapi/yar": "10.1.1",
-    "applicationinsights": "2.3.3",
-    "govuk-frontend": "4.4.0",
-    "joi": "17.7.0",
+    "@hapi/hapi": "21.3.2",
+    "@hapi/hoek": "11.0.2",
+    "@hapi/inert": "7.1.0",
+    "@hapi/vision": "7.0.3",
+    "@hapi/wreck": "18.0.1",
+    "@hapi/yar": "11.0.1",
+    "applicationinsights": "2.7.3",
+    "govuk-frontend": "4.7.0",
+    "joi": "^17.10.1",
     "moment": "2.29.4",
-    "nunjucks": "3.2.3"
+    "nunjucks": "^3.2.4"
   },
   "devDependencies": {
-    "@hapi/good": "9.0.0",
-    "@hapi/good-console": "9.0.0",
+    "@hapi/good": "9.0.1",
+    "@hapi/good-console": "9.0.1",
     "@hapi/good-squeeze": "6.0.0",
```

Comment:
`good` has been deprecated so longer term might be "good" to swap for `hapi-pino` or similar

Reply:
Swapped good to Hapi-pino